build-tests/api-extractor-scenarios/etc/includeForgottenExports/api-extractor-scenarios.api.json

```diff
@@ -0,0 +1,550 @@
+            },
+            {
+              "kind": "Content",
+              "text": ";"
+            }
+          ],
+          "returnTypeTokenRange": {
+            "startIndex": 1,
+            "endIndex": 2
+          },
+          "releaseTag": "Public",
+          "overloadIndex": 1,
+          "parameters": [],
+          "name": "someFunction4"
+        },
+        {
+          "kind": "Function",
+          "canonicalReference": "api-extractor-scenarios!someFunction5:function(1)",
+          "docComment": "/**\n * @public\n */\n",
+          "excerptTokens": [
+            {
+              "kind": "Content",
+              "text": "export declare function someFunction5(): "
+            },
+            {
+              "kind": "Content",
+              "text": "internal2."
+            },
+            {
+              "kind": "Reference",
+              "text": "ForgottenExport6",
+              "canonicalReference": "api-extractor-scenarios!ForgottenExport6:class"
+            },
+            {
+              "kind": "Content",
+              "text": ";"
+            }
+          ],
+          "returnTypeTokenRange": {
+            "startIndex": 1,
+            "endIndex": 3
+          },
+          "releaseTag": "Public",
+          "overloadIndex": 1,
+          "parameters": [],
+          "name": "someFunction5"
+        },
+        {
+          "kind": "Namespace",
+          "canonicalReference": "api-extractor-scenarios!SomeNamespace1:namespace",
+          "docComment": "/**\n * @public\n */\n",
+          "excerptTokens": [
+            {
+              "kind": "Content",
+              "text": "export declare namespace SomeNamespace1 "
+            }
+          ],
+          "releaseTag": "Public",
+          "name": "SomeNamespace1",
+          "preserveMemberOrder": false,
+          "members": [
+            {
+              "kind": "Class",
+              "canonicalReference": "api-extractor-scenarios!SomeNamespace1.ForgottenExport3:class",
+              "docComment": "",
+              "excerptTokens": [
+                {
+                  "kind": "Content",
+                  "text": "class ForgottenExport3 "
+                }
+              ],
+              "releaseTag": "Public",
+              "name": "ForgottenExport3",
+              "preserveMemberOrder": false,
+              "members": [],
+              "implementsTokenRanges": []
+            },
+            {
+              "kind": "Function",
+              "canonicalReference": "api-extractor-scenarios!SomeNamespace1.someFunction3:function(1)",
+              "docComment": "",
+              "excerptTokens": [
+                {
+                  "kind": "Content",
+                  "text": "export function someFunction3(): "
+                },
+                {
+                  "kind": "Reference",
+                  "text": "ForgottenExport3",
+                  "canonicalReference": "api-extractor-scenarios!~ForgottenExport3:class"
```

Comment:
Notice that the canonical reference here is missing the namespace. I believe this is an existing bug in `DeclarationReferenceGenerator`.

Reply:
Opened a fix at https://github.com/microsoft/rushstack/pull/3566.